main.py

```diff
@@ -53,20 +53,28 @@ def prepare_metadata(train_data, valid_data, test_data, public_paths):
 @action.command()
 @click.option('-d', '--dev_mode', help='if true only a small sample of data will be used', is_flag=True, required=False)
 def prepare_masks(dev_mode):
-    if params.erode_selem_size > 0:
-        erode = params.erode_selem_size
-        target_dir = params.masks_overlayed_eroded_dir
+    erode = eval(params.erosion_percents)
+    if erode != [0]:
```

Comment:
@apyskir I'd prefer erode[0] = 0 but I don't exactly know (or can show example) why

Reply:
@jakubczakon You can imagine, that you want to prepare masks, that are not eroded, that are 10% and 20% eroded, so you pass a list [0,10,20]. Then your condition will prepare only not eroded masks, what is not what you want. I'll leave it as it is, ok?